pkg/bridgemodel/utils.go

```diff
@@ -68,7 +68,7 @@ func (t *HttpClient) sendAuthorizedRequest(method string, url string, body []byt
 	return
 }
 
-func ConfigureDefaultTransport() {
+func ConfigureDefaultTransportToSkipTlsValidation() {
```

Comment:
This function wasn't actually used so I renamed it and made use of it since it did exactly what I needed it to do.